TrailBlaze/Networking/PaceImprovementTracking.h

```diff
@@ -7,22 +7,26 @@
 
 #import <Foundation/Foundation.h>
 #import "CoreLocation/CoreLocation.h"
+#import "CoreMotion/CoreMotion.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
+
 @interface PaceImprovementTracking : NSObject
```

Comment:
i'd recommend naming this `PaceImprovementTracker` to re-enforce that it's a concrete class.

the `-ing` suffix is very commonly used for protocol naming. from [apple guidelines](https://www.swift.org/documentation/api-design-guidelines/#naming):

> Protocols that describe a capability should be named using the suffixes able, ible, or ing (e.g. Equatable, ProgressReporting).